spec/controllers/api/devices/devices_controller_index_spec.rb

```diff
@@ -4,21 +4,21 @@
 
   include Devise::TestHelpers
 
-  describe '#index' do
+  describe '#show' do
 
     let(:user) { FactoryGirl.create(:user) }
 
     it 'returns all the users devices, as JSON' do
       sign_in user
-      get :index, format: :json
-      device = user.devices.first
-      id = JSON.parse(response.body).first["_id"]
+      get :show, format: :json
+      device = user.device
+      id = JSON.parse(response.body)["_id"]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.